doc/METADATA.ini

```diff
@@ -1295,3 +1295,14 @@ status = implemented
 usedby/plugins = cache
 type = boolean
 description = tells the cache plugin to remove all cache files during the next `kdbGet` in a safe way.
+
+[tomltype]
```

Comment:
I would prefer if it is directly integrated in the types of Elektra (type, enum, array) and only for the tables there is some internal/toml/type or we even add the "tables" to standardtypes. I would prefer the name "section" though (similar to array as extra meta-data).